### PR TITLE
Add Tails backup tutorial

### PR DESCRIPTION
**Pull request by RiccardoMasutti**

### CODE REVIEW

**Comment by yahiheb** [comment]
NACK.
Isn't it easier to simply put the link https://tails.boum.org/doc/first_steps/persistence/copy/index.en.html
somewhere rather than copy all that info in this repository.

**Comment by RiccardoMasutti** [comment]
@yahiheb Sure we can do that, maybe by adding a big red warning on backup importance and link to tails.boum.org.

What do you think about it? @MaxHillebrand

**Comment by PulpCattel** [comment]
There's already a _:::danger_ at the end, we could add everything there.

:::danger
Remember to backup either your files or your [persistent storage](https://tails.boum.org/doc/first_steps/persistence/copy/index.en.html)!
:::

File: docs/building-wasabi/Credits.md [hunk 1]
```diff
@@ -17,3 +17,4 @@ Here are some credits, thanks and sources (in casual order):
 - [Bitcoin.org](https://bitcoin.org/en/)
 - [Bitcoin GitHub repository](https://github.com/bitcoin/)
 - [CryptoSeb, Cryptopaper](https://github.com/cryptoseb/CryptoPaper)
+- [Tails](https://tails.boum.org/)
```
Comment (yahiheb) [inline docs/building-wasabi/Credits.md]:
```suggestion
```

This is not necessary anymore.

**Pull request merged**